kits/java/src/main/java/com/luxai/lux/utils/FactoryProcessor.java

```diff
@@ -0,0 +1,23 @@
+package com.luxai.lux.utils;
+
+import com.luxai.lux.Environment;
+import com.luxai.lux.Factory;
+import com.luxai.lux.Obs;
+import com.luxai.lux.action.UnitAction;
+
+import java.util.Map;
+
+public class FactoryProcessor {

```

Comment:
This class appears to be part of the agent logic, right? I don't think that belongs in the `lux` subpackage.

Reply:
fix